### PR TITLE
Svelte examples in the documentation

### PR DESCRIPTION
**Pull request by j3rem1e**
Issue:

## What I did

Updated the documentation to includes Svelte snippets

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@j3rem1e thank you very much for the time and effort you put into this and get the Svelte examples included in the documentation. Let me take a look and see if everything is ok and we'll get this merged. Do you mind waiting a bit? 


Stay safe

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@j3rem1e first and foremost thank you for the time and effort you put into the pull request. This is really great. I left some items that require attention and once you've addressed this we should be able to merge this and get svelte snippets into the documentation. 

Let me know.

Stay safe

File: docs/writing-docs/doc-blocks.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ This is extremely useful, but it can be further expanded. Additional information
     'react/button-component-with-proptypes.ts.mdx',
     'angular/button-component-with-proptypes.ts.mdx',
     'vue/button-component-with-proptypes.vue.mdx'
+    'svelte/button-component-with-proptypes.js.mdx',
```
[the remark this answers is not in the document]
Comment (j3rem1e) [reply docs/writing-docs/doc-blocks.md:30]: oh my bad. I had rebased my PR but failed to merge it apparently.. I will fix it

File: docs/snippets/svelte/button-group-story.js.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const Template = (args) => ({
+  Component: ButtonGroup,
+  props: args,
+ });
+
+export const Pair = Template.bind({});
+Pair.args = {
+  buttons: [ ...ButtonStories.Primary.args, ...ButtonStories.Secondary.args ],
+  orientation: 'horizontal',
```
[the remark this answers is not in the document]
Comment (j3rem1e) [reply docs/snippets/svelte/button-group-story.js.mdx:20]: what do you mean by 'there is an error in the console ?' I don't know where to look at :)

This code come from the react example, but you're right, it's not good.

**Review by shilman — APPROVED** [review]
❤️ thank you @j3rem1e !

**Pull request merged**